app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_post_comment
+  def update_post_comment
+    posts.increment!(:comments_counter)
+  end
```

Comment:
- [ ] Well done using after_save  to call the #update_post_comment  method every time a new post has been saved to database, I just recommend to make the update_post_comment method private.